### PR TITLE
C# LINQ Element Operator Sample

### PR DESCRIPTION
**Pull request by matthewreily**
Added some samples for Element Operators #134 cc @BethMassi  @cartermp

### CODE REVIEW

File: samples/linq/csharp/element/FirstSample-2.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
```
Comment (BethMassi) [inline samples/linq/csharp/element/FirstSample-2.cs]: Unnecessary using

File: samples/linq/csharp/element/Program.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+public class Program
```
Comment (BethMassi) [inline samples/linq/csharp/element/Program.cs]: Should probably be consistent with other samples and put this in a namespace

**Comment by cartermp** [comment]
LGTM!  Let's rebase and then I'll merge.

**Comment by matthewreily** [comment]
@cartermp Not sure if I did it right.  I messed up previously and created a pull request from my master branch.  If it's not right now, I'll close this request and submit a fresh pull request using a proper branch this time.

**Comment by cartermp** [comment]
If that's easier for you then that's fine.  We can just link back to this PR discussion for posterity's sake.

**Pull request closed without merging**